src/ImageSharp/Formats/Webp/Lossless/LosslessUtils.cs

```diff
@@ -122,66 +128,57 @@ public static void AddGreenToBlueAndRed(Span<uint> pixelData)
             if (Avx2.IsSupported)
             {
                 int numPixels = pixelData.Length;
-                fixed (uint* p = pixelData)
+                int i;
+                for (i = 0; i + 8 <= numPixels; i += 8)
```

Comment:
```suggestion
                for (i = 0; i <= numPixels - 8; i += 8)
```
So the value for `numPixels - 8` can be kept in a register. With `i + 8` it must be re-calculated for each iteration.